pkg/dependencymagnet/docs.go

```diff
@@ -0,0 +1,11 @@
+//go:build tools
```

Comment:
Could we consider naming this pkg/openshift/magnet.go?

Just want to make it clear that this is not part of normal upstream.

Reply:
ok